snakemake_executor_plugin_googlebatch/command.py

```diff
@@ -31,38 +31,31 @@
 )
 
 install_snakemake = """
-# Only the main job should install conda (rest can use it)
 echo "I am batch index ${BATCH_TASK_INDEX}"
 export PATH=/opt/conda/bin:${PATH}
-if [ $BATCH_TASK_INDEX = 0 ] && [ ! -d "/opt/conda" ] ; then
-    workdir=$(pwd)
-    url=https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    wget ${url} -O ./miniconda.sh
-    chmod +x ./miniconda.sh
-    bash ./miniconda.sh -b -u -p /opt/conda
-    rm -rf ./miniconda.sh
-    conda config --system --set channel_priority strict
-    which python
-    /opt/conda/bin/python --version
-    url=https://github.com/snakemake/snakemake-interface-common
-    git clone --depth 1 ${url} /tmp/snakemake-common
-    cd /tmp/snakemake-common
-    /opt/conda/bin/python -m pip install .
-    url=https://github.com/snakemake/snakemake-interface-executor-plugins
-    git clone --depth 1 ${url} /tmp/snakemake-plugin
-    cd /tmp/snakemake-plugin
-    /opt/conda/bin/python -m pip install .
-    git clone --depth 1 https://github.com/snakemake/snakemake /tmp/snakemake
-    cd /tmp/snakemake
-    /opt/conda/bin/python -m pip install .
-    cd ${workdir}
-fi
+repo=https://raw.githubusercontent.com/snakemake/snakemake-executor-plugin-googlebatch
+path=add-preemtible/scripts/install-snek.sh
+wget ${repo}/${path}
+chmod +x ./install-snek.sh
+workdir=$(pwd)
+url=https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+wget ${url} -O ./miniconda.sh
+chmod +x ./miniconda.sh
+bash ./miniconda.sh -b -u -p /opt/conda
+rm -rf ./miniconda.sh
+
+which python
+/opt/conda/bin/python --version
+./install-snek.sh https://github.com/snakemake/snakemake-storage-plugin-gcs
```

Comment:
It does not, at least when I tested. I would say (since the credits expired and I can't test right now) we leave for now, because we know this works, and I can likely use some of my credits to test again without, and if it works without I'll remove in a follow up PR (with the next feature).

Reply:
Ok!

Reply:
It however means that we are bound to gcs again. While this is the most common use case, the executor is independent of the storage plugin though. Hence, as soon as possible, we should fix the auto-deployment.